modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -99,8 +119,21 @@ object Target extends WithGid('t'.refined) with TargetOptics {
         Sidereal.TrackOrder.compare(a, b)
       case (a @ Nonsidereal(_, _, _), b @ Nonsidereal(_, _, _)) =>
         Nonsidereal.TrackOrder.compare(a, b)
-      case (Nonsidereal(_, _, _), _)                            => -1
-      case _                                                    => 1
+      case (a @ Opportunity(_, _, _), b @ Opportunity(_, _, _)) =>
+        Opportunity.RegionOrder.compare(a, b)
+
+      // Nonsidereal sorts first
```

Comment:
Just curious. is there a reason for this ordering of NonSidereal, Sidereal, Opportunity?